src/gasPriceOracle/adapters/op.ts

```diff
@@ -0,0 +1,17 @@
+import assert from "assert";
+import { BigNumber, providers } from "ethers";
+import { chainIsOPStack } from "../../utils";
+import { GasPriceEstimate } from "../types";
+
+// 50 Wei should be sufficient to land a transaction on the OP stack.
```

Comment:
How confident are you in this number? Curious how you got it

Reply:
Discussed w/ @cc7768. We could also scale it up to like 1k Wei with ~0 consequence.